.travis.yml

```diff
@@ -17,7 +17,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
 
 script:
-  - make all
+  - BASEURL=http://127.0.0.1:9500; make -e all
```

Comment:
To simplify, we can assume that `make all` is a target meant for travis. That way you would not need to change anything.